### DISCUSSION TITLE
[Desktop] ComboBox does not react on MouseWheel

### DISCUSSION THREAD

**Discussion by kucint** [unoplatform/uno#17489] [Q&A] [answered]
In WinUI3, `ComboBox` reacts on MouseWheel, when control has focus.
Unfortunately, in `Desktop` MouseWheel is not working.

**Comment by morning4coffe-dev** [comment]
Hey @kucint, thanks for pointing this out. I was able to reproduce the issue with both the latest stable and dev versions. I've created an issue to track it here: https://github.com/unoplatform/uno/issues/17491

**Reply by morning4coffe-dev** [reply, marked as answer]
Update: fix for this is now on its way in PR https://github.com/unoplatform/uno/pull/17493
